ros/processor/insights_engine_result_consumer.py

```diff
@@ -115,7 +128,47 @@ def process_report(self, host, reports):
                     fqdn=host['fqdn'],
                     rule_hit_details=reports,
                     number_of_recommendations=rec_count,
-                    state=SYSTEM_STATES[state_key]
+                    state=SYSTEM_STATES[state_key],
+                    instance_type=performance_record.get('instance_type')
+                )
+                LOG.info(
+                    f"{self.prefix} - System created/updated successfully: {host['id']}"
+                )
+
+                # For Optimized state, reports would be empty, but utilization_info would be present
+                if reports:
+                    set_default_utilization = True if reports[0].get('key') == 'NO_PCP_DATA' else False
+                else:
+                    set_default_utilization = False
+
+                if set_default_utilization is False:
+                    performance_utilization = {
+                        'memory': utilization_info['mem_utilization'],
+                        'cpu': utilization_info['cpu_utilization'],
+                        'io': convert_iops_from_percentage(utilization_info['io_utilization'])
+                    }
+                    # Adding max_io to utilization dict
+                    performance_utilization.update(
+                       {'max_io': max(performance_utilization['io'].values())}
+                    )
+                else:
+                    LOG.info(f"{self.prefix} - Setting default utilization for performance profile")
```

Comment:
```suggestion
                    LOG.debug(f"{self.prefix} - Setting default utilization for performance profile")
```
Do we really want to log this information with level `info`?